js/models.js

```diff
@@ -365,12 +365,17 @@ models.service('models', ['$rootScope', '$filter', 'bufferResume', function($roo
             });
         }
 
+        var prefixtext = "";
+        for (var pti = 0; pti < prefix.length; ++pti) {
+            prefixtext += prefix[pti].text;
+        }
+
         var rtext = "";
         for (var i = 0; i < content.length; ++i) {
             rtext += content[i].text;
         }
 
-       return {
+        return {
```

Comment:
Oh it was wrong before. Sorry, didn't catch that. Thanks for fixing in that case :)